### PR TITLE
Movie updater

### PR DESCRIPTION
**Pull request by pcota**

### CODE REVIEW

**Comment by pcota** [comment]
@Mmdixon for review as well

**Comment by freeone3000** [comment]
Looks good to me.

File: docker-movie-updater/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ENV LC_ALL en_US.UTF-8
+
+ADD https://s3.amazonaws.com/aws-cli/awscli-bundle.zip /tmp/
+RUN unzip /tmp/awscli-bundle.zip -d /tmp \
+	&& /tmp/awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
+
+ADD start-movie-updater.sh /opt/start-movie-updater.sh
+RUN chmod +x /opt/start-movie-updater.sh
+
+# Set the time zone to the local time zone
```
Comment (Mmdixon) [inline docker-movie-updater/Dockerfile]: local for who? Us, the server, or where westworld is?

File: docker-movie-updater/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Building
+
+To build the image, simply invoke
+
+    docker build github.com/maluuba/docker-tomcat7
+
+A prebuilt container is also available in the docker index
+
+    docker pull maluuba/tomcat7
```
Comment (Mmdixon) [inline docker-movie-updater/README.md]: This README looks like the tomcat7 one, not the movie-updater

**Pull request merged**